app/controllers/food_controller.rb

```diff
@@ -0,0 +1,10 @@
+class FoodController < ApplicationController
+    def index
+      @food = food.all
```

Comment:
The model name should be capitalized as it is stated in the food.rb file

Reply:
This probelm is already solved kindely check latest commits